src/client/user/components/CreateUrlModal/styles/createLinkForm.ts

```diff
@@ -2,9 +2,9 @@ import { createStyles, makeStyles } from '@material-ui/core'
 
 type LinkFormStyles = {
   textFieldHeight: number
-  isFile: boolean
-  uploadFileError: string | null
-  createShortLinkError: string | null
+  isFile?: boolean
+  uploadFileError?: string | null
+  createShortLinkError?: string | null
```

Comment:
why is the properties being made optional here? were they wrong before? don't we always have `isFile` present?

Reply:
This is the styling that is used by both CreateLinkForm and FormStartAdorment. 

So, CreateLinkForm requires all the props' options but FormStartAdorment only need one, thus the properties are optional. They do not change the domain logic but just the styling for the respective components. 

We can separate the styles but we will have more, additional lines of code for slightly different styling

Reply:
I think the actual code smell here is that the styling naming is being coupled to the URL domain. I don't think a styling file should have to know about uploading files or creating short links.

Reply:
Managed to decouple the styles for FormStartAdorment and CreateLinkForm.

However, I think styling here fits the requirement because it is changing real-time based on the toggling between the file and url tab. Though I am not sure what you mean by the styling naming?